### PR TITLE
add config option for controlling 24 hours

### PR DESCRIPTION
**Pull request by danyalsiraj**

### CODE REVIEW

**Comment by danyalsiraj** [comment]
Thanks for reviewing @PeideAmonNg , can it be merged?

**Comment by PeideAmonNg** [comment]
@danyalsiraj I don't have write access to merge.

File: lib/templates/bootstrap/datepicker.android.js [hunk 1]
```diff
@@ -85,6 +85,11 @@ function datepicker(locals) {
     <Text style={dateValueStyle}>{formattedValue}</Text>
   ) : null;
 
+  var is24Hour = true;
```
Comment (alvaromb) [inline lib/templates/bootstrap/datepicker.android.js]: Please use `let`.

File: lib/templates/bootstrap/datepicker.android.js [hunk 2]
```diff
@@ -85,6 +85,11 @@ function datepicker(locals) {
...
+  if (typeof locals.config.is24Hour !== "undefined") {
```
Comment (alvaromb) [inline lib/templates/bootstrap/datepicker.android.js]: Why not `locals.config.is24Hour` directly?

Also, `is24Hour` should be added into the docs.

Comment (alvaromb) [reply lib/templates/bootstrap/datepicker.android.js]: I see, the check is not inside https://github.com/danyalsiraj/tcomb-form-native/blob/9df6e4fcc9711e9e1d3668d694720366f18ad8fd/lib/templates/bootstrap/datepicker.android.js#L54

Can you move it there in order to match the coding style?

**Comment by bingle87** [comment]
This is either not fixed or I do not know how to set it.

`var options = {
    auto: 'none', // do not generate automatic labels nor automatic placeholders
    fields: {
        time: {
            mode: 'time',
            is24Hour: false,
            config: {
              is24Hour: false,
              format: (date) => moment(date).format('LT'),
            },
        },
    },
  };`

Can anyone help me understand. Thanks.

**Pull request closed without merging**